pkg/sqlmigrations/migrations.go

```diff
@@ -1587,25 +1592,29 @@ func createRoleOptionsTable(ctx context.Context, r runner) error {
 	return nil
 }
 
-func addCreateRoleToAdminAndRoot(ctx context.Context, r runner) error {
+func addOptionToAdminAndRoot(ctx context.Context, r runner, option string) error {
 	// Upsert the admin/root roles with CreateRole privilege into the table.
 	// We intentionally override any existing entry.
 	const upsertCreateRoleStmt = `
-          UPSERT INTO system.role_options (username, option, value) VALUES ($1, 'CREATEROLE', NULL)
+          UPSERT INTO system.role_options (username, option, value) VALUES ($1, $2, NULL)
```

Comment:
Should there be validation here that `option` is one of the allowed strings? Or does the `role_options` table have a check constraint to ensure this?